src/user/user-info.inferface.ts

```diff
@@ -0,0 +1,5 @@
+export interface MinimalUserInfo {
```

Comment:
~`Info` 붙이는 대신 `MinimalUser`로 심플하게 가면 어떨까요?
```suggestion
export interface MinimalUser {
```